README.md

```diff
@@ -18,7 +18,7 @@ composing asynchronous and event-based programs using observable sequences
 that are exposed as asynchronous streams, expanding on the
 [observer pattern](https://en.wikipedia.org/wiki/Observer_pattern),
 strongly inspired by [ReactiveX](http://reactivex.io/),
-but designed from the ground up  for back-pressure and made to cleanly interact
+but designed from the ground up  for back-pressure and made to cleanly iterant
```

Comment:
:-)) yeah, freaking IntelliJ IDEA, it's a bad idea to "search in comments and strings"